src/itemdef.cpp

```diff
@@ -254,6 +254,8 @@ class CItemDefManager: public IWritableItemDefManager
 			palette(NULL)
 		{}
 
+		ClientCached(const ClientCached &other) = delete;
```

Comment:
Please use `DISABLE_CLASS_COPY`, otherwise there'll still be an implicitly defined copy assignment operator.